src/components/Filters/FilterDescription.js

```diff
@@ -12,15 +16,43 @@ const ButtonWrapper = styled.button`
   padding: 3px 8px;
 `
 
+const getOperatorText = (operator, type, schemaEntry) => {
+  const operatorItem = PineTypes[type].rules[operator]
+  if (isPlainObject(PineTypes[type].rules[operator]) && schemaEntry) {
```

Comment:
NIT: I guess that this could be `isPlainObject(operatorItem)`.

Reply:
oops